<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CConfigurationWizard</class>
 <widget class="QWizard" name="CConfigurationWizard">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>swift configuration wizard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ClassicStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HaveCustomButton1</set>
  </property>
  <widget class="swift::gui::components::CLegalInfoWizardPage" name="wp_Legal">
   <property name="title">
    <string>Legal</string>
   </property>
   <property name="subTitle">
    <string>Please confirm you know and understand our license and privacy policy</string>
   </property>
   <layout class="QVBoxLayout" name="vl_Legal">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="swift::gui::components::CLegalInfoComponent" name="comp_LegalInformation"/>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CInitialDataLoadWizardPage" name="wp_DataLoad">
   <property name="title">
    <string>Data loading</string>
   </property>
   <property name="subTitle">
    <string>Caches model and ICAO data</string>
   </property>
   <layout class="QVBoxLayout" name="vl_DataLoad">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="swift::gui::components::CInitialDataLoadComponent" name="comp_DataLoad">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CCopyModelsFromOtherSwiftVersionsWizardPage" name="wp_CopyModels">
   <property name="title">
    <string>Copy models</string>
   </property>
   <property name="subTitle">
    <string>Copy model data from other swift versions. Here you can copy your model set or model cache from another swift version.</string>
   </property>
   <layout class="QVBoxLayout" name="vl_CopyModels">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="swift::gui::components::CCopyModelsFromOtherSwiftVersionsComponent" name="comp_CopyModels"/>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CCopySettingsAndCachesWizardPage" name="wp_CopySettingsAndCaches">
   <property name="title">
    <string>Copy settings and caches</string>
   </property>
   <property name="subTitle">
    <string>Copy settings and caches from other swift versions</string>
   </property>
   <layout class="QVBoxLayout" name="vl_CopySettingsAndCache">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QScrollArea" name="sa_CopySettingsAndCache">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="wi_CopySettingsAndCache">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>90</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="vl_CopySettingsAndCacheWidget">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="swift::gui::components::CCopySettingsAndCachesComponent" name="comp_CopySettingsAndCachesComponent"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CConfigSimulatorWizardPage" name="wp_Simulator">
   <property name="title">
    <string>Simulator</string>
   </property>
   <property name="subTitle">
    <string>Simulator(s) you are using</string>
   </property>
   <layout class="QVBoxLayout" name="vl_SimulatorSetup">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="swift::gui::components::CConfigSimulatorComponent" name="comp_Simulator">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>125</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CFirstModelSetWizardPage" name="wp_FirstModelSet">
   <property name="title">
    <string>First model set</string>
   </property>
   <property name="subTitle">
    <string>Create a simple model set (for a start)</string>
   </property>
   <layout class="QVBoxLayout" name="vl_FirstModelSet">
    <item alignment="Qt::AlignRight">
     <widget class="QPushButton" name="pb_SkipFirstModelSet">
      <property name="text">
       <string>  I do this later / do not need this  </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="swift::gui::components::CFirstModelSetComponent" name="comp_FirstModelSet">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>175</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CInstallSimulatorSpecificWizardPage" name="wp_SimulatorSpecific">
   <property name="title">
    <string>Simulator specific installations</string>
   </property>
   <property name="subTitle">
    <string>Install xswiftbus (X-Plane only) or swift terrain probe (FSX/P3D only)</string>
   </property>
   <layout class="QVBoxLayout" name="vl_SimulatorSpecific">
    <item>
     <widget class="QToolBox" name="tb_SimulatorSpecific">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pg_InstallFsxTerrainProbe">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>89</width>
         <height>64</height>
        </rect>
       </property>
       <attribute name="label">
        <string extracomment="FSX/P3D: Needed with the x86 legacy 32bit SimConnect">Install FSX terrain probe</string>
       </attribute>
       <layout class="QVBoxLayout" name="vl_P3DFsx">
        <item alignment="Qt::AlignTop">
         <widget class="swift::gui::components::CInstallFsxTerrainProbeComponent" name="comp_InstallFsxTerrainProbe">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pg_InstallXSwiftBus">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>75</width>
         <height>64</height>
        </rect>
       </property>
       <attribute name="label">
        <string extracomment="Install the xswiftbus component">XPlane: Install xswiftbus</string>
       </attribute>
       <layout class="QVBoxLayout" name="vl_XPlane">
        <item alignment="Qt::AlignTop">
         <widget class="swift::gui::components::CInstallXSwiftBusComponent" name="comp_InstallXSwiftBus">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="swift::gui::components::CConfigHotkeyWizardPage" name="wp_Hotkeys">
   <property name="title">
    <string>Hotkeys</string>
   </property>
   <property name="subTitle">
    <string>Configure (at least) your PTT key</string>
   </property>
   <layout class="QVBoxLayout" name="vl_Hotkeys">
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <widget class="QGroupBox" name="gb_Hotkeys">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Hotkeys</string>
      </property>
      <layout class="QVBoxLayout" name="vl_GroupBoxHotkey">
       <item>
        <widget class="swift::gui::components::CSettingsHotkeyComponent" name="comp_Hotkeys"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::components::CFirstModelSetComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/firstmodelsetcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CConfigSimulatorComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/configsimulatorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CConfigSimulatorWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/configsimulatorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInitialDataLoadWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/initialdataloadcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInitialDataLoadComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/initialdataloadcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CSettingsHotkeyComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/settingshotkeycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CConfigHotkeyWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/settingshotkeycomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInstallXSwiftBusComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/installxswiftbuscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CFirstModelSetWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/firstmodelsetcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCopyModelsFromOtherSwiftVersionsWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/copymodelsfromotherswiftversionscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCopyModelsFromOtherSwiftVersionsComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/copymodelsfromotherswiftversionscomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CLegalInfoComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/legalinfocomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CLegalInfoWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/legalinfocomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCopySettingsAndCachesComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/copysettingsandcachescomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CCopySettingsAndCachesWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/copysettingsandcachescomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInstallSimulatorSpecificWizardPage</class>
   <extends>QWizardPage</extends>
   <header>gui/components/installsimulatorspecificwizardpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>swift::gui::components::CInstallFsxTerrainProbeComponent</class>
   <extends>QFrame</extends>
   <header>gui/components/installfsxterrainprobecomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pb_SkipFirstModelSet</sender>
   <signal>clicked()</signal>
   <receiver>CConfigurationWizard</receiver>
   <slot>next()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
